src/components/Navbar.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import '../App.css';
+import { MdPerson } from 'react-icons/md';
 
 const Navbar = () => (
   <nav>
-    <h1>Bookstore CMS</h1>
+    <h1 className="header">Bookstore CMS</h1>
```

Comment:
- As Requested by your last reviewer [here](https://github.com/stephenkati/book-store/pull/5#discussion_r1189882217), Kindly make sure to follow the styles from the [zeplin](https://pedantic-beaver-cb923b.netlify.app/#/) one to make it look exactly the same. I think it would be better to stick to sizes, margins, and the font-family provided

**Zeplin Design**
![Screenshot from 2023-05-10 15-36-19](https://github.com/stephenkati/book-store/assets/97834160/b4fbeab3-6b9e-40ef-93cd-9a303436c443)

**Your Design** 
![Screenshot from 2023-05-10 15-34-19](https://github.com/stephenkati/book-store/assets/97834160/95796408-9c5f-4ebd-bde1-68ecfcd75833)

Reply:
I see, on it